components/UI/Button.js

```diff
@@ -0,0 +1,32 @@
+import styled from 'styled-components';
+
+const Button = styled.button`
+  display: grid;
+  grid-auto-flow: column;
+  place-content: center;
+  padding: calc(1rem * 0.25) calc(1rem * 0.5);
+  font: inherit;
+  text-transform: uppercase;
+  cursor: pointer;
+  border-style: solid;
+  border-width: var(--border-width);
+  border-radius: var(--border-radius);
+  appearance: none;
+
+  svg {
+    fill: var(--color-background) !important;
```

Comment:
Something feels wrong about the `!important`...
Is there no way of avoiding this?

Reply:
When taking all of the `!important`s away I end up with this:
<img width="590" alt="image" src="https://github.com/medienhaus/medienhaus-spaces/assets/706419/64604bd3-b6a4-4669-a22b-7e4676606494">

... which would be fine if we were to discourage using just the `<Button>` component.

Reply:
At some point during development and testing, I had problems with this line: https://github.com/medienhaus/medienhaus-spaces/blob/main/components/UI/Icon.js#L8

Hence, the `!important`; I don’t think I really consider it harmful within a styled component, though we should remove it, if everything works as intended. Maybe the reason why I initially added it is not even a convern anymore after the plenty last commits. ✨